docs/source/guide/prompts_overview.md

```diff
@@ -0,0 +1,105 @@
+
+
+
+## Prompt evaluation and fine-tuning
+
+!!! info Tip
+    While you can use Prompts to do basic prompt evaluation and fine-tuning, we recommend using [Evals](evals_overview), which has been specifically designed for this purpose and has much more robust analytics. 
+
+As you evaluate your prompt against the ground truth annotations, we return an accuracy score for each version of your prompt. You can use this to iterate your prompt versions for [clarity, specificity, and context](prompts_draft#Drafting-effective-prompts). 
+
+![Screenshot of accuracy score](/images/prompts/accuracy_score.png)
+
+This accuracy score provides a measurable way to evaluate and refine the performance of your prompt. By tracking accuracy, you can ensure that the automated labels generated by the LLM are consistent with ground truth data. 
+
+This feedback loop allows you to iteratively fine-tune your prompts, optimizing the accuracy of predictions and enhancing the overall reliability of your data annotation processes. In industries where data accuracy directly impacts decision-making and operational efficiency, this capability is invaluable.
+
+## Model types
+
+### Text classification 
+
+At present, the Prompts feature only supports text classification labeling tasks.  
+
+Text classification is the process of assigning predefined categories or labels to segments of text based on their content. This involves analyzing the text and determining which category or label best describes its subject, sentiment, or purpose. The goal is to organize and categorize textual data in a way that makes it easier to analyze, search, and utilize. 
+
+Text classification labeling tasks are fundamental in many applications, enabling efficient data organization, improving searchability, and providing valuable insights through data analysis. Some examples include:
+
+* **Spam Detection**: Classifying emails as "spam" or "not spam."
+* **Sentiment Analysis**: Categorizing user reviews as "positive," "negative," or "neutral."
+* **Topic Categorization**: Assigning articles to categories like "politics," "sports," "technology," etc.
+* **Support Ticket Classification**: Labeling customer support tickets based on the issue type, such as "billing," "technical support," or "account management."
+* **Content Moderation**: Identifying and labeling inappropriate content on social media platforms, such as "offensive language," "hate speech," or "harassment."
+
+
+## Workflow
```

Comment:
do we need a workflow for the bootstrapping mode? the only steps that diverge are 2 (GT not needed), 4 (no evaluate step) and 5 (no accuracy, just vibe check)